src/usr/bin/brightness-reset

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import wx
+import subprocess
+from os import system
+
+class BrightnessRestore(wx.Frame):
+
+    def detect_display_devices(self):
+        """Detects available displays"""
+        connected_devs = []
+        xrandr_output = subprocess.check_output('xrandr -q', shell=True)
+        lines = xrandr_output.split('\n')
+        for line in lines:
+            words = line.split(' ')
+            for word in words:
+                if word == 'connected':
+                    connected_devs.append(words[0])
+        return connected_devs
+
+    def __init__(self):
+        self.detected_devices = self.detect_display_devices()
+	self.primary_name = self.detected_devices[0]
+	cmd1 = "xrandr --output \
```

Comment:
At first I tried to restore brightness of both displays (cmd1 &cmd2), but it showed some errors when no secondary display was connected. I forgot to remove 1 from cmd1.